### PR TITLE
Adding ability to name your VPN config on import instead of having it…

### PR DESCRIPTION
**Pull request by phutchins**
… show up as your username

### CODE REVIEW

**Comment by flaccid** [comment]
@phutchins do you think it might be better to name the attribute something like `client_prefix` instead?
Also, will need a commit for the update of `README.md`.

**Comment by phutchins** [comment]
@flaccid sure, that makes sense to me. PR updated to reflect the attr name change as well as README.

File: attributes/default.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@
 default['openvpn']['key']['message_digest'] = 'sha256'
 
 # Cookbook attributes
+default['openvpn']['client_prefix']     = 'vpn-prod'
```
Comment (flaccid) [inline attributes/default.rb]: Need to remove extra spacing to line up the assignments.

**Comment by flaccid** [comment]
Just need to remove some extra spacing to make rubocop happy please.

**Comment by phutchins** [comment]
@flaccid Done. Not sure how that extra space snuck in there. :)

**Comment by flaccid** [comment]
All good.

**Pull request merged**